src/components/data-management/SpeciesSelector.jsx

```diff
@@ -21,6 +21,8 @@ const SpeciesSelector = (props) => {
       dropdownMatchSelectWidth={400}
       showSearch
       placeholder='Search for common or scientific name...'
+      data-test-class='species-select'
+      data-test-id={testID}
```

Comment:
I think it's better to directly use the value `fill-species-select` here instead of passing it via a props. Is the going to change?

Reply:
Yes, I need the selector on the fill species box to be different from the ones beside samples. This is ugly so I wouldn't mind a better way to distinguish them

Reply:
Do you need to distinguish between different species selectors? If you don't then you don't need to use `data-test-id`. Using data-test-class is fine. You can query for `.species-select` then select a specific element using `.eq()`

Reply:
Yes I do. I suppose I could try to get the "Fill species" box and get the selector within that. Using `eq` seems super fragile imo

Reply:
I think using `eq()` to select an element and modify it is not fragile, as long as you can ensure that the correct selecteors are applied and you control the data that is being tested. I use it in `randomizeSampleName()` to select the project name and change it. I see that you are using `Int Test - Medata Project` in your PR:  https://github.com/biomage-ltd/testing/pull/15/files, so I think you can always be confident that the data that is loaded in the project will be there.